src/test/java/api/put/UpdateListTest.java

```diff
@@ -20,7 +21,7 @@ public class UpdateListTest extends BaseTest {
     private String idList;
 
     @BeforeMethod
-    @Description("Create board and list")
+    @Step("Create board and list")
     public void createBoardAndList() {
         idBoard = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
         idList = listTestRestClient.createList(constructDefaultListKeyValue(), idBoard).getId();
```

Comment:
make it look like this 

@BeforeMethod
@Step("Test preparation")
 public void preconditions() {
       idBoard = boardRestTestClient.createNewBoard(constructDefaultBoardKeyValue()).getId();
       idList = listTestRestClient.createList(constructDefaultListKeyValue(), idBoard).getId();
    }